### PR TITLE
Fix line and column fields in tokens.

### PR DESCRIPTION
**Pull request by sangisos**
Also adds one test case for column which fails before fix. Reverts old hack with variable tabsize. Can be fixed in separate issue. 

Updates #14.
Replaces #19.

### CODE REVIEW

File: test/t1/t1_test.go [hunk 1]
```diff
@@ -58,3 +59,18 @@ func Test2(t *testing.T) {
 		}
 	}
 }
+
+func Test3(t *testing.T) {
+	toks := lexer.NewLexer([]byte(`c b`))
```
Comment (awalterschulze) [inline test/t1/t1_test.go]: So there error here is at the EOF, right?

**Comment by awalterschulze** [comment]
Looks good.  
I would like to test this on github.com/katydid/katydid and github.com/awalterschulze/gographviz
I hope to get some time to it, this weekend, I hope thats not too long to wait.

**Comment by sangisos** [comment]
The error should be at the b, because according to the grammar, the only
member of the followset for c is the EOF.

Med Vänliga Hälsningar
Alexander Andersson
On 18 Oct 2016 07:42, "Walter Schulze" <notifications@github.com> wrote:

*@awalterschulze* commented on this pull request.
------------------------------

In test/t1/t1_test.go
<https://github.com/goccmack/gocc/pull/30#pullrequestreview-4604193>:

> @@ -58,3 +59,18 @@ func Test2(t *testing.T) {
 		}
 	}
 }
+
+func Test3(t *testing.T) {
+	toks := lexer.NewLexer([]byte(`c b`))

So there error here is at the EOF, right?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/goccmack/gocc/pull/30#pullrequestreview-4604193>, or mute
the thread

**Comment by awalterschulze** [comment]
Ok so we are counting 1,2,3 not 0,1,2
fair enough :)

**Comment by awalterschulze** [comment]
tested it and its definitely better :)
+1

**Comment by awalterschulze** [comment]
I want to merge this.
Any objections?

**Comment by sangisos** [comment]
No objections. Maybe I should add the field DefaultTabsize and set it to 1,
but I can do that in an separete issue/pull request.

Med Vänliga Hälsningar
Alexander Andersson

**Comment by mewmew** [comment]
> I want to merge this.
> Any objections?

Go for it!

**Comment by awalterschulze** [comment]
Thank you @sangisos for your efforts :)

**Pull request merged**